File: Hackerrenk/new_year_chaos.cpp
void minimumBribes(vector<int> q) {
    int count=0;
    bool ans= true;
    int n= q.size();

    for(int i=0;i<n;i++){
        if(q[i] > i+3){
            ans=false;
        }
    }

    for(int i=0;i<n;i++){
        if(q[i]!=i+1){
            for(int j=i+1;j<n;j++){
                if(q[j] == i+1){
                    for(int k=j;k>=i+1;k--){
                        q[k]=q[k-1];
                        count++;
                    }
                    q[i] =i+1;
                    break;


                }
            }
        }
    }
    if(ans){
        cout<<count<<"\n";
    
    }
    else{
        cout<<"Too chaotic"<<"\n";
    }
}

File: Hackerrenk/new_year_chaos.py
def minimumBribes(q):
    count=0
    for i in range(len(q)-1, 0, -1):
        if(q[i]!= i +1):
            if(q[i-1] == i+1):
                count += 1
                q[i], q[i-1] = q[i-1], q[i]
             
            elif(q[i-2] == i+1):
                count += 2
                q[i-2], q[i-1] = q[i-1], q[i-2]
                q[i-1], q[i] = q[i], q[i-1]
                
            else:
                print("Too chaotic")
                return()
    print(count)
    return()
